code/modules/clothing/suits/armor.dm

```diff
@@ -229,7 +229,7 @@
 			if(M == owner)
 				continue
 			owner.Beam(M,icon_state="lightning[rand(1, 12)]",icon='icons/effects/effects.dmi',time=5)
-			M.adjustFireLoss(25)
+			M.adjustFireLoss(10)
```

Comment:
"Tesla" armor should use the electrocute_act() proc with tesla_shock var active, instead of direct damage application, in my opinion.